### PR TITLE
Fix css changes and add patternfly changes

### LINKED ISSUE #1036: Use patternfly components throughout the installer app
Based on the guidelines and uniformity across different apps, use of `Pattern Fly` is recommended. Will be using the features for the UX components in accordance with that.

### PR DESCRIPTION
**Pull request by mohitsuman**
Fix #1036:

1. Change the use of components to use `patternfly` and then apply custom css.
2. Update the `grid column` style of css.
3. Add UX for login for account.html page.

For row elements of selection page, added border to distinguish between them based on cursor movement. Attached video.

giphy

### CODE REVIEW

**Comment by dgolovin** [comment]
test this

**Comment by dgolovin** [comment]
image
Checkbox is not in the middle as it was(?) before.

File: browser/app.css [hunk 1]
```diff
@@ -937,4 +931,19 @@ footer {
     margin-right: 10px;
     float: right;
   }
+
+  .panel-group-items .panel:not(.zero-border):hover {
+    border-left: 3px solid #c00;
```
Comment (dgolovin) [inline browser/app.css]: border-left with 3px wide makes component card moves right 1px when mouse hovers over. We should ether have it 2px, or figure out how to make card stay on the same place when mouse hoovers over it.

**Comment by dgolovin** [comment]
All not primary buttons (Back and Cancel) should look like secondary buttons from patternfly examples.

File: browser/app.css (hunk 1 above)
Comment (mohitsuman) [reply browser/app.css]: Fixed in https://github.com/redhat-developer-tooling/developer-platform-install/pull/1062/commits/6c04fe5849b2cc6e6cba9eb6257d1a40212922e4

**Comment by mohitsuman** [comment]
The checkbox is also aligned at the centre with the change in padding.  
https://github.com/redhat-developer-tooling/developer-platform-install/pull/1062/commits/6c04fe5849b2cc6e6cba9eb6257d1a40212922e4#diff-7cc3025cbe6f198e4624ce63d1c75636R597

**Pull request merged**